### DISCUSSION TITLE
NCSOFT Launcher (Purple) log in black screen...

### DISCUSSION THREAD

**Discussion by martin5618** [Whisky-App/Whisky#724] [Q&A]
Hey there,

I'm trying to install the Purple Launcher from NCSOFT. It seems to work so far, but when I'm trying to log in, a separate web window opens. However, it's not displayed correctly, as shown in screenshot 1.

<img width="1296" alt="Screenshot 1" src="https://github.com/Whisky-App/Whisky/assets/154507103/4ddae739-6b64-475b-8c89-caf71587cdf3">

You can still log in by clicking and typing in the right positions, but after that, the login buffering won't stop.

I'm not a pro in this software mechanics, but maybe it has something to do with any missing winetricks. I've installed MS .NET 4.8 and Visual C++ 2015-2022 so far. Maybe I missed something.

I would really appreciate it if someone could help me with this.

🙏

**Comment by Ravenaige** [comment]
I have same issue

you fixed?

**Comment by smurfwizard** [comment]
I have the same issue, is there any fix?
